pyenv-win/libexec/pyenv-install.vbs

```diff
@@ -283,17 +301,18 @@ Sub extract(params, register)
             End If
         End If
     ElseIf params(LV_Web) Then
-        exitCode = deepExtract(params)
+        exitCode = deepExtract(params, True)
     ElseIf objfs.GetExtensionName(installFile) = "zip" Then
         exitCode = unzip(installFile, installPath, zipRootDir)
     Else
-        Dim quiet
-        Dim dev
+        exitCode = deepExtract(params, False)
+        ' Dim quiet
+        ' Dim dev
 
-        If params(IP_Quiet) Then quiet = " /quiet"
-        If params(IP_Dev) Then dev = " Include_debug=1 Include_symbols=1 Include_dev=1 "
+        ' If params(IP_Quiet) Then quiet = " /quiet"
+        ' If params(IP_Dev) Then dev = " Include_debug=1 Include_symbols=1 Include_dev=1 "
 
-        exitCode = objws.Run(qInstallFile & quiet & dev &" InstallAllUsers=0 Include_launcher=0 Include_test=0 SimpleInstall=1 TargetDir="& qInstallPath, 9, True)
+        ' exitCode = objws.Run(qInstallFile & quiet & dev &" InstallAllUsers=0 Include_launcher=0 Include_test=0 SimpleInstall=1 TargetDir="& qInstallPath, 9, True)
```

Comment:
See the comment above, now `pyenv install 3.xxx --dev` is not installing the debug symbols.